website/addons/wiki/static/wikiPage.js

```diff
@@ -0,0 +1,245 @@
+'use strict';
+var ko = require('knockout');
+var $ = require('jquery');
+var $osf = require('osfHelpers');
+
+var mathrender = require('mathrender');
+var md = require('markdown').full;
+var mdQuick = require('markdown').quick;
+var diffTool = require('wikiDiff');
```

Comment:
Perhaps we should rename `wikiDiff.js` to `diffTool.js` to make it more generic.